<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>map_menu</class>
 <widget class="QTabWidget" name="map_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>100</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QTabWidget::pane { 

border-top: 2px solid rgb(78, 125, 224);
background: rgb(66, 66, 66);
}
QTabWidget::tab-bar {
    background-color: rgb(0, 0, 0);
}
QTabBar::tab {
    color:rgb(255, 255, 255);
    background: rgb(66, 66, 66);
    min-width: 8ex;
    padding: 6px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    border-top: 2px solid rgb(78, 125, 224);
    color: rgb(78, 125, 224);
}

QTabBar::tab:selected {
    
    border-color: rgb(66, 66, 66);
    border-top: 2px solid rgb(78, 125, 224);
}</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="action">
   <attribute name="title">
    <string>Action</string>
   </attribute>
   <widget class="QPushButton" name="action_left">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/left.png);
}

QPushButton:pressed {
border-image: url(:/left_press.png);
}

QPushButton::checked {
border-image: url(:/left_press.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="action_right">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>15</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/right.png);
}

QPushButton:pressed {
border-image: url(:/right_press.png);
}

QPushButton::checked {
border-image: url(:/right_press.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="action_forward">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>15</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/up.png);
}

QPushButton:pressed {
border-image: url(:/up_press.png);
}

QPushButton::checked {
border-image: url(:/up_press.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="action_back">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>15</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/down.png);
}

QPushButton:pressed {
border-image: url(:/down_press.png);
}

QPushButton::checked {
border-image: url(:/down_press.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="navigation">
   <attribute name="title">
    <string>Navigation</string>
   </attribute>
   <widget class="QPushButton" name="PatrolCTL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgb(78, 125, 224);
}

QPushButton:pressed {
    background-color: rgb(28, 59, 124);
}

QPushButton::checked {
   background-color: rgb(28, 59, 124);
}</string>
    </property>
    <property name="text">
     <string>Patrol
off</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clearGoals">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>15</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgb(78, 125, 224);
}

QPushButton:pressed {
    background-color: rgb(28, 59, 124);
}

QPushButton::checked {
   background-color: rgb(28, 59, 124);
}</string>
    </property>
    <property name="text">
     <string>Clear
Goals</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Nav_NavigationMode">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>15</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgb(78, 125, 224);
}

QPushButton:pressed {
    background-color: rgb(28, 59, 124);
}

QPushButton::checked {
   background-color: rgb(28, 59, 124);
}</string>
    </property>
    <property name="text">
     <string>Navigation</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Nav_ExploreMode">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>15</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgb(78, 125, 224);
}

QPushButton:pressed {
    background-color: rgb(28, 59, 124);
}

QPushButton::checked {
   background-color: rgb(28, 59, 124);
}</string>
    </property>
    <property name="text">
     <string>Explore</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="slam">
   <attribute name="title">
    <string>Slam</string>
   </attribute>
   <widget class="QPushButton" name="slam_savemap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgb(78, 125, 224);
}

QPushButton:pressed {
    background-color: rgb(28, 59, 124);
}

QPushButton::checked {
   background-color: rgb(28, 59, 124);
}</string>
    </property>
    <property name="text">
     <string>Save Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slam_clearMap">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>15</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgb(78, 125, 224);
}

QPushButton:pressed {
    background-color: rgb(28, 59, 124);
}

QPushButton::checked {
   background-color: rgb(28, 59, 124);
}</string>
    </property>
    <property name="text">
     <string>Clear
Map</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="setting">
   <attribute name="title">
    <string>Setting</string>
   </attribute>
   <widget class="QLineEdit" name="lineEdit_IP">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>5</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid rgb(78, 125, 224);  
    border-radius: 3px; 
    background: white;  
    selection-background-color: green;
    font-size: 14px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue; 
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Port">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>35</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid rgb(78, 125, 224);  
    border-radius: 3px; 
    background: white;  
    selection-background-color: green;
    font-size: 14px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue; 
}</string>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(78, 125, 224);</string>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(78, 125, 224);</string>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
